### PR TITLE
Gdb 6329 handle properly license expired

### PR DESCRIPTION
**Pull request by desislava-hristova-ontotext**

### CODE REVIEW

**Comment by desislava-hristova-ontotext** [comment]
Jenkins retry build

**Comment by desislava-hristova-ontotext** [comment]
Jenkins retry build

**Comment by desislava-hristova-ontotext** [comment]
Jenkins retry build

File: src/js/angular/core/templates/core-errors.html [hunk 1]
```diff
@@ -5,12 +5,13 @@
                 <div class="alert alert-warning lead">
                     <div ng-if="!getActiveRepository()" >Some functionalities are not available because you are not connected to any repository.</div>
                     <div id="restrictedDiv" ng-if="getActiveRepository() && isRestricted">Some functionalities are not available because
+                        <span ng-if="!isLicenseValid()">your license is not valid.</span>
                         <span ng-if="isSecurityEnabled() && !canWriteActiveRepo()">you have no write permission to repository </span>
-                        <strong ng-if="isActiveRepoOntopType()">{{getActiveRepository()}}</strong>
-                        <span ng-if="canWriteActiveRepo() && isActiveRepoOntopType()"> is read-only Virtual Repository</span>
-                        <span ng-if="canWriteActiveRepo() && isActiveRepoFedXType()"> <b>{{title}}</b> view is not supported by FedX Repository</span>.</div>
+                        <strong ng-if="isActiveRepoOntopType()">{{getActiveRepository()}}.</strong>
```
[the remark this answers is not in the document]
Comment (desislava-hristova-ontotext) [reply src/js/angular/core/templates/core-errors.html]: Fixed

**Review by sava-savov-ontotext — APPROVED** [review]

**Pull request merged**